document_page_approval_levels/models/document_page.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+############################################################################
+#    Module Writen For Odoo, Open Source Management Solution
+#
+#    Copyright (c) 206 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+#    coded by: Luis Torres <luis_t@vauxoo.com>
+#    planned by: Sabrina Romero <sabrina@vauxoo.com>
+############################################################################
+from openerp import models, fields, api
+
+
+class DocumentPage(models.Model):
+    """This class is use to manage Document."""
+    _inherit = 'document.page'
+
+    approver_items_id = fields.One2many(
+        'approver.document.page.item', 'document_id', 'Approver items')
+
+    def create_history(self, page_id, content):
+        """Inherit method to add news items by each approver in document."""
+        item_obj = self.env['approver.document.page.item']
+        res = super(DocumentPage, self).create_history(page_id, content)
+        for item in self.browse(page_id).approver_items_id:
+            item_obj.create({
+                'user_id': item.user_id.id,
+                'history_id': res.id,
+            })
+        return res
+
+    @api.multi
+    def write(self, vals):
+        """Added that update approvers in histories related to this
+        document when this change."""
```

Comment:
I prefer a server action instead a overwritting of the write method, can you use that apporach please, ask @JulioSerna for better understanding of that approach.